src/main/java/seedu/address/logic/parser/UntagCommandParser.java

```diff
@@ -14,7 +14,7 @@
 /**
  * Parses input arguments and creates a new UntagCommand object
  */
-public class UntagCommandParser {
+public class UntagCommandParser implements Parser<UntagCommand> {
```

Comment:
Just an opinion: Technically, UntagCommandParser and TagCommandParser, the only difference is the Type of Command it returns right?

@mynameizzhafeez any thoughts?

Reply:
Yea we could have just one parser.

Reply:
We could possibly use Enums to denote whether to Tag/Untag, like what i did with MeetCommand.

Reply:
EduMateParser could then initial use the Parser with one of these enums.

Reply:
For example new TagCommandParser(TagType tagType).